src/platform/datapath_winuser.c

```diff
@@ -269,11 +236,6 @@ typedef struct CXPLAT_SEND_DATA {
     //
     uint16_t SegmentSize;
 
-    //
-    // The type of ECN markings needed for send.
-    //
-    uint8_t ECN; // CXPLAT_ECN_TYPE
```

Comment:
Does ECN really need to be in the common?

Reply:
ECN seems to be used in both normal socket and xdp.